tests/sql/test_sql.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+import pytest
+
+import daft
+from daft.sql.sql import SQLCatalog
```

Comment:
nit: is there any way we can have this just `from daft.sql`  instead of `from daft.sql.sql